mlir/tools/rocmlir-tuning-driver/rocmlir-tuning-driver.cpp

```diff
@@ -89,6 +92,31 @@ static llvm::cl::opt<rock::TuningParamSetKind> tuningSpaceKind(
     llvm::cl::value_desc("tuning space to use"),
     llvm::cl::init(rock::TuningParamSetKind::Full));
 
+static llvm::cl::opt<unsigned> numIterations(
+    "num-iterations",
+    llvm::cl::desc("Number of times to run each kernel for averaging"),
+    llvm::cl::value_desc("number of runs"), llvm::cl::init(100));
+
+static llvm::cl::opt<unsigned> trimPercent(
+    "trim-percent",
+    llvm::cl::desc("Percentage to trim from top and bottom of results"),
+    llvm::cl::value_desc("trim percentage"), llvm::cl::init(10));
+
+static llvm::cl::opt<unsigned> sleepMs(
+    "sleep-ms",
+    llvm::cl::desc("Milliseconds to sleep between runs to avoid throttling"),
+    llvm::cl::value_desc("milliseconds to sleep"), llvm::cl::init(0));
+
+static llvm::cl::opt<unsigned>
+    sleepFreq("sleep-freq", llvm::cl::desc("Sleep after every N runs"),
+              llvm::cl::value_desc("number of runs to sleep after"),
+              llvm::cl::init(10));
```

Comment:
Every 10 might be too frequent IMO and can slow down tuning process. 
I was thinking it would be be good to sleep after benchmarking each kernel and not inbetween the kernel itself.